migration/migrator/data/course_tables.sql

```diff
@@ -1062,7 +1062,7 @@ CREATE TABLE public.users (
     last_updated timestamp(6) with time zone,
     time_zone character varying DEFAULT 'NOT_SET/NOT_SET'::character varying NOT NULL,
     display_image_state character varying DEFAULT 'system'::character varying NOT NULL,
-    registration_subsection character varying(255),
+    registration_subsection character varying(255) DEFAULT (NOT NULL::boolean),
```

Comment:
For some reason when I ran the course dump, it set the default to NOT NULL::boolean. Do you know why this would be happening @MasterOdin ?

Reply:
It's because your alter table statement isn't right. With postgres, you can only change one aspect of a column at a time, and if you want to change multiple things, you need multiple alters. By doing `SET DEFAULT '' NOT NULL`, postgres is doing a weird transformation of `'' NOT NULL` into a singular default value which ends up being `(NOT NULL::boolean)`.

I've updated my suggestion below with what I think the command should be, though did not test against actual DB.

Reply:
With your latest change, my guess would be that you were getting a weird syntax quirk of using the single quotes to define your python string literal as well as default value inside that string. I made a small commit that seems to have fixed this and I am getting the expected result on dumping the migrated schema.